vm-setup/roles/v1aX_integration_test/tasks/inspection.yml

```diff
@@ -0,0 +1,50 @@
+---
+  - name: Wait until "{{ NUM_NODES }}" bmhs become ready.
```

Comment:
This might look odd that we are using shell in Ansible :), but unfortunately field selector doesn't work for CRDs it seems, so the only way I can see it to rely on jq.
It would be nice to have something like this to filter out ready bmh, but that's not possible it seems right now. This not the limitation of Ansible, but rather Kubernetes. Similar discussion in [BMO](https://github.com/metal3-io/baremetal-operator/issues/779). 
```
- name: Search for all running pods
  community.kubernetes.k8s_info:
    kind: Pod
    field_selectors:
      - status.phase=Running
```

Reply:
This is done also in this PR. https://github.com/metal3-io/metal3-dev-env/pull/623. Depending on which one lands first, this would be redundant I suppose ?

Reply:
Right. I think this can go first. I was interested to test it in 623, and that's why I had to add it there too. But I will anyways need to rebase 623, once this will get merged.